streams/src/main/java/org/apache/kafka/streams/kstream/internals/suppress/KTableSuppressProcessor.java

```diff
@@ -55,10 +65,14 @@ public void close() {
 
     @Override
     public String toString() {
-        return "KTableSuppressProcessor{suppress=" + suppress + '}';
+        return "KTableSuppressProcessor{" +
+            "suppress=" + suppress +
+            ", keySerde=" + keySerde +
+            ", valueSerde=" + valueSerde +
+            '}';
     }
 
-    static class NotImplementedException extends RuntimeException {
+    public static class NotImplementedException extends RuntimeException {
```

Comment:
Just so we can assert that it happens in the scenario tests (which are in a different package)